custom/enikshay/templates/enikshay/partials/report_content.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
```

Comment:
this is because of https://github.com/dimagi/commcare-hq/pull/13748 right? @calellowitz do you have any other ideas?

if this is the only solution that's dissatisfying but I think we can just move forwards with it. we should probably at least put a comment indicating the situation and reason and linking to the PR